collections/luci-nginx/root/etc/uci-defaults/60-luci

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+chmod +x /etc/init.d/luci
```

Comment:
Why do we need this? We could install this file with the right mods during installation in the Makefile